temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -229,3 +230,28 @@ message GetRegionResponse {
     // The temporal cloud region.
     temporal.api.cloud.region.v1.Region region = 1;
 }
+
+message GetAccountRequest {
```

Comment:
Do we ever expect that a user can view/edit users from another account? Often in these cases you want to accept an identifier and just default it to the current user account. This allows you to prepare for a future general purpose API. You can still provide two HTTP endpoints, one for identifiable accounts resources and one for current.

Reply:
I think there is still value for consistency sake to accept an identifier here and expose an identifier on `Account`. The identifier here can default to the current user's.

Reply:
I am thinking we can easily add the account-id later when we need to. Something like this: 
https://github.com/temporalio/api-cloud/compare/abhinav/accountManagement...abhinav/accountManagement-acctIDExt => adds a account-id to a few endpoints.

Reply:
You definitely _can_ wait, I was just saying I think the consistency across resources has value now. At the very least even if not adding to the getter, maybe add an `id` to `Account`.

Reply:
> maybe add an id to Account

This has had a lot of conversations on the cloud team. Over the last 2 years we have made a lot of effort to hide the account-id from the user. This was one of the product requirements, "a customer should not need to deal with their account ids...", there is some legacy reasons associated to it.

Reply:
I think we should differentiate hiding from a standard user and hiding from a developer. If we want to hide from both, ok, but I disagree and I think an account identifier (even if you can't imagine how it could be used today) is a normal thing to have.

Reply:
Added id to the account as suggested.

Reply:
I think it would make sense for this operation to accept an account ID, but we can table it for the future. If set now, it can be defaulted today to requiring it either be unset or set as the current caller's account ID.

Reply:
I am inclining to adding it in the future. Might create a little unnecessary confusion with what to use for this optional field.